.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,56 @@
+name: Bug report
+description: Create a report to help us improve
+labels: ["bug", "needs triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Please make sure to fill out the entire form below, providing as much context as you can in order to help us triage and track down your bug as quickly as possible.
+
+        Before filing a bug, please be sure you have searched through [existing bugs](https://github.com/signalfx/splunk-otel-collector-chart/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Abug) to see if an existing issue covers your bug.
+  - type: textarea
+    attributes:
+      label: What happened?
+      description: Please provide as much detail as you reasonably can.
+      value: |
+        ## Description
+
+        ## Steps to Reproduce
+
+        ## Expected Result
+
+        ## Actual Result
+
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Chart version
+      description: What version did you use? (e.g., `v0.4.0`, `1eb551b`, etc)
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Environment information
+      description: Please provide any additional information about your installation.
+      value: |
+        ## Environment
+        Cloud: (e.g., "EKS", "AKS", "GKE")
+        k8s version: (e.g., 1.21.0)
+        OS: (e.g., "Ubuntu 20.04")
+
+  - type: textarea
+    attributes:
+      label: Chart configuration
+      description: Please provide the configuration you are using (e.g. the values.yaml config file).
```

Comment:
suggest we put in a psa reminding users to redact access tokens

Reply:
https://github.com/signalfx/splunk-otel-collector-chart/pull/731 is open